docs/GettingStarted.md

```diff
@@ -65,7 +65,8 @@ of the Datadog tracer, you can override the following defaults:
       debug: false,
       trace_agent_hostname: 'localhost',
       trace_agent_port: 8126,
-      env: Rails.env
+      env: Rails.env,
+      tags: {}
```

Comment:
@ufoot I don't like having both, it seems we can provide two different paths to do the same thing.